.github/workflows/ci.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       matrix:
         # Python 3.8 is the last non-EOL version. Also adapt tox.ini
-        python-version: ['3.8', 'pypy3.10', '3.12']
+        python-version: ['3.8', 'pypy3.10', '3.x']
```

Comment:
I prefer to push users to not build on tools that no longer receive security updates.
If they want to use EOL versions of Python, pip will just install an earlier version of cpplint.

Reply:
1. There's a fairly big bug (namespace indentation false positives) that will be fixed in the next release. I'd prefer we drop 3.8 after the next release.
2. I don't like taking commit authorship for others' ideas.